--- git/configure	2017-02-10 22:19:01.000000000 +0100
+++ git.new/configure	2017-02-10 22:37:30.256208165 +0100
@@ -3358,12 +3358,12 @@
 case $_backend in
 	androidsdl | dingux | gph | linuxmoto | maemo | openpandora | samsungtv | sdl)
 		find_sdlconfig
-		append_var INCLUDES "`$_sdlconfig --prefix="$_sdlpath" --cflags`"
-		append_var LIBS "`$_sdlconfig --prefix="$_sdlpath" --libs`"
+		append_var INCLUDES "`pkg-config sdl --cflags`"
+		append_var LIBS "`pkg-config sdl --libs`"
 		append_var DEFINES "-DSDL_BACKEND"
 		add_line_to_config_mk "SDL_BACKEND = 1"
 
-		_sdlversion=`$_sdlconfig --version`
+		_sdlversion=`pkg-config sdl --modversion`
 		case $_sdlversion in
 			2.0.*)
 				add_line_to_config_mk "USE_SDL2 = 1"
@@ -4441,8 +4441,8 @@
 	if test -z "$_freetypeconfig"; then
 		_freetype2=no
 	else
-		FREETYPE2_LIBS=`$_freetypeconfig --prefix="$_freetypepath" --libs`
-		FREETYPE2_CFLAGS=`$_freetypeconfig --prefix="$_freetypepath" --cflags`
+		FREETYPE2_LIBS=`pkg-config freetype2 --libs`
+		FREETYPE2_CFLAGS=`pkg-config freetype2  --cflags`
 
 		if test "$_freetype2" = "auto"; then
 			_freetype2=no
