From b47800f17e81d35178c28503d02ab0c31141b0ef Mon Sep 17 00:00:00 2001
From: reichi <stephan@reichholf.net>
Date: Wed, 20 Feb 2019 11:30:06 +0100
Subject: [PATCH] add dreambox platform

---
 Makefile.libretro | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/Makefile.libretro b/Makefile.libretro
index 3f30866..7c6cde3 100644
--- a/Makefile.libretro
+++ b/Makefile.libretro
@@ -45,6 +45,12 @@ else ifeq ($(platform), unix)
    SHARED :=  -shared -Wl,--version-script=$(CORE_DIR)/libretro/core/link.T -Wl,--no-undefined
    PLATFLAGS := -DALIGN_DWORD
 
+else ifeq ($(platform), dreambox)
+   TARGET := $(TARGET_NAME)_libretro.so
+   fpic := -fPIC
+   SHARED :=  -shared -Wl,--version-script=$(CORE_DIR)/libretro/core/link.T -Wl,--no-undefined
+   PLATFLAGS := -DALIGN_DWORD
+
 # Classic Platforms ####################
 # Platform affix = classic_<ISA>_<µARCH>
 # Help at https://modmyclassic.com/comp
-- 
2.17.1

